policies/technical/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: OpenSSL Technical Policies
+subtitle: |
+    First issued 30th September 2020
+    <br />
+    Moved to GitHub Nov 2021
+---
+These are the technical policies and procedures established by the OTC
+in accordance with the [project bylaws].
+
+<p>
+    <ul>
+        <!--#include virtual="index.inc" -->
+    </ul>
+</p>
+
+They are maintained in the [openssl/technical-policies] repository on GitHub.
+
+
+[project bylaws]: ../omc-bylaws.html
```

Comment:
Would this be better referencing the glossary?

[project bylaws]: ../glossary.html#bylaws

Reply:
....  so any link will have to go through the glossary?  Not sure I agree with the forced indirection

Reply:
I can, though, if you insist

Reply:
This was the only one that stood out as possibly benefiting from the indirection.  I'm not going to insist however.

The side bar will still have a direct link I believe.